notebooks/image_models/solutions/object_detection_automl_vertex.ipynb

```diff
@@ -0,0 +1,427 @@
+{
```

Comment:
<ul><li>Maybe outlining the steps needed to get a prediction may help to remember / understand the flow: e.g. something like: Step 1 - encode the image, Step 2 - create the instance, Step 3 - create the parameters (why?), Step 4 - Request the prediction.</li><li>Add maybe a brief description of what the <code>base64</code> call does and why they are needed, along with a short explanation about the call to <code>.decode("utf-8")</code></li></ul><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>